tests/Routing/RoutingRouteTest.php

```diff
@@ -1403,8 +1403,12 @@ public function testJsonResponseIsReturned()
         });
 
         $response = $router->dispatch(Request::create('foo/bar', 'GET'));
-        $this->assertNotInstanceOf(\Illuminate\Http\Response::class, $response);
-        $this->assertInstanceOf(\Illuminate\Http\JsonResponse::class, $response);
+
+        $response->setEncodingOptions(0);
+
+        $this->assertInstanceOf(\Illuminate\Http\Response::class, $response);
+        $this->assertInstanceOf(\Illuminate\Http\JsonResponse::class, $response->baseResponse);
+        $this->assertEquals(0, $response->getEncodingOptions());
```

Comment:
`$this->assertEquals($response->getData(true), $response->baseResponse->getData(true));`fails unfortunatelly